addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2245,16 +2245,14 @@ export class OdooEditor extends EventTarget {
     }
 
     /**
-     * Returns true if the current selection is inside the editable.
+     * Returns true if the current selection is inside the editable and collapsed.
      *
      * @private
      * @param {Object} selection
      * @returns {boolean}
      */
     _isSelectionInEditable(selection) {
-        return !selection.isCollapsed &&
-            this.editable.contains(selection.anchorNode) &&
-            this.editable.contains(selection.focusNode);
+        return (selection.isCollapsed && this.editable.contains(selection.anchorNode)) || (!selection.isCollapsed && this.editable.contains(selection.anchorNode) && this.editable.contains(selection.focusNode));
```

Comment:
I expected to get:
```js
return  this.editable.contains(selection.anchorNode) &&
            this.editable.contains(selection.focusNode);
```
Did I miss something ? What is the point of testing both cases ?

Reply:
Indeed, overlook from my part.